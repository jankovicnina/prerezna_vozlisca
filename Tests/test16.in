581 1202
534 70 6.23
381 71 8.09
406 335 -1.09
40 279 2.18
238 509 2.52
403 219 -0.29
225 506 7.25
557 139 8.33
5 329 3.9
189 228 4.05
537 505 -0.27
146 46 8.01
308 303 5.76
396 516 3.65
567 168 -1.95
331 293 5.76
233 251 2.8
207 19 5.93
453 75 4.23
562 522 6.13
191 7 6.08
250 272 -0.54
245 191 9.82
564 518 9.14
279 380 1.97
112 553 4.1
235 474 6.94
125 505 7.76
100 527 -0.62
178 124 3.54
326 399 -0.13
525 533 4.73
353 300 0.7
504 61 5.09
485 98 1.91
86 170 4.77
511 26 2.38
68 276 1.94
449 411 9.1
340 323 2.53
343 294 5.5
574 337 5.08
549 36 -0.79
513 30 9.87
302 554 4.02
437 531 5.07
29 55 8.41
446 400 0.17
101 229 2.54
539 29 9.97
235 110 5.54
221 228 9.26
48 151 7.11
64 152 3.9
355 417 0.8
350 304 6.54
339 71 1.17
63 570 7.85
289 570 0.41
26 247 3.81
486 366 0.19
457 198 3.75
36 427 4.35
553 269 5.2
245 146 2.16
374 483 9.09
428 338 3.56
141 232 9.07
548 422 8.22
464 143 7.47
379 89 -1.84
249 445 1.67
346 2 -0.44
565 366 5.76
18 541 7.79
335 366 2.74
303 25 6.14
544 313 7.62
404 534 8.38
117 498 0.14
144 24 7.75
189 470 9.22
540 240 3.68
326 562 3.99
424 310 7.92
85 468 -1.69
377 159 9.38
139 120 5.85
373 242 -1.27
71 492 5.9
380 432 7.39
553 308 3.06
122 174 -1.59
397 57 1.51
317 501 -1.8
163 185 6.86
550 251 9.33
265 187 7.39
128 257 4.17
471 103 0.7
318 92 5.94
340 469 4.97
451 541 3.16
559 534 2.56
483 213 0.13
519 99 -1.17
106 323 7.19
300 128 0.82
349 315 0.16
265 364 -1.91
45 31 4.28
89 402 6.8
350 127 -1.79
4 76 8.41
569 88 7.92
496 244 -0.66
184 370 3.1
209 447 6.33
12 356 2.81
428 328 -0.1
114 294 6.02
480 356 8.62
214 144 3.33
206 526 0.31
271 116 6.48
219 333 8.73
499 535 5.34
63 181 6.02
454 550 -0.61
382 286 7.0
256 246 8.59
516 28 3.23
430 490 7.01
178 33 8.28
309 220 7.23
575 426 0.68
198 309 5.78
492 295 2.28
348 384 -1.26
497 127 3.93
426 253 1.51
1 23 1.67
151 347 1.92
510 155 4.07
141 206 9.44
563 408 2.86
267 317 2.64
66 186 -0.42
487 512 0.34
239 140 4.3
129 278 3.91
58 538 1.91
395 205 -1.74
157 517 6.45
0 124 3.95
336 136 5.79
511 526 0.72
422 104 -0.74
45 137 1.05
389 295 9.8
416 290 8.7
375 271 6.03
531 463 -0.74
260 502 0.07
426 562 0.07
178 61 5.3
299 19 4.6
459 197 9.3
509 179 7.91
9 566 7.18
514 400 0.34
429 232 6.43
550 214 0.15
6 114 0.77
43 531 -0.51
228 576 1.6
310 74 7.97
436 301 2.67
550 130 -0.41
358 456 7.11
247 425 6.03
57 9 3.08
491 35 9.28
407 478 7.28
441 314 1.82
192 69 0.7
199 493 6.29
181 255 -0.93
53 122 1.78
477 264 6.83
421 341 -1.52
22 227 1.43
340 477 -0.89
162 450 3.81
539 470 8.07
544 341 -0.6
44 335 3.55
413 444 9.5
234 386 2.34
229 237 6.09
221 37 7.61
458 357 -0.04
126 257 8.28
488 440 8.25
148 54 0.8
169 457 1.05
312 509 9.38
217 245 4.25
234 577 6.17
370 431 -1.11
72 25 8.18
493 106 4.32
572 524 -1.17
136 532 6.51
274 239 8.2
109 165 5.6
244 364 2.2
550 461 1.23
468 379 -0.17
445 214 -0.0
330 496 7.85
214 189 4.99
520 154 5.05
387 50 -0.09
148 85 6.68
301 503 9.0
322 371 7.61
160 49 -0.24
335 365 6.42
277 334 5.87
49 512 8.46
29 17 9.19
32 109 7.51
422 555 1.42
35 274 7.85
147 140 8.38
104 465 -0.67
70 123 3.03
83 112 2.86
551 41 9.94
362 176 4.34
266 224 5.3
34 307 1.33
569 537 7.96
138 71 0.5
561 210 6.45
254 224 -1.21
156 141 7.97
193 172 4.38
204 101 1.82
447 397 9.12
312 144 1.48
339 299 -1.52
461 267 8.27
198 426 3.6
297 507 8.97
494 135 8.78
475 311 6.5
379 168 3.04
75 556 5.61
136 147 9.24
213 414 4.52
185 65 0.42
222 542 0.87
140 294 1.42
166 9 6.88
312 17 -1.03
82 387 -1.98
272 320 2.1
545 371 3.97
526 248 2.42
380 368 4.32
551 116 0.66
425 460 1.53
113 412 2.93
520 88 9.76
243 178 0.99
33 34 9.58
75 206 4.86
360 403 0.17
440 22 9.66
417 350 5.71
237 374 6.75
256 13 0.01
443 462 -1.06
192 361 6.51
39 156 0.49
118 248 6.44
31 576 7.79
260 410 5.84
536 87 8.15
254 53 9.92
128 183 4.31
7 545 2.17
491 305 1.01
341 258 7.72
215 423 3.62
247 34 2.36
175 446 6.64
313 206 0.38
58 226 1.73
217 441 1.55
248 188 3.36
78 482 8.41
547 239 8.79
148 241 7.99
142 98 5.16
146 516 3.01
134 11 7.35
392 278 9.93
166 1 6.88
337 477 5.97
158 6 0.51
392 366 7.94
495 331 -0.11
249 346 2.96
145 525 5.35
8 282 3.78
155 258 2.31
62 180 2.15
255 397 -0.06
347 84 5.26
291 202 -1.21
461 213 -0.36
1 330 9.6
274 498 -0.82
60 312 4.29
329 399 2.48
393 17 6.96
257 125 9.44
110 481 0.5
425 15 0.74
171 362 1.78
561 332 5.38
101 129 6.58
211 163 -0.91
467 369 2.75
507 435 8.2
409 540 9.53
391 34 2.04
497 138 3.95
429 549 6.83
93 225 9.6
363 150 7.78
143 81 7.1
327 357 7.23
375 424 8.48
72 378 2.57
573 547 6.88
258 18 1.05
103 546 2.33
345 46 5.16
446 430 -1.76
269 84 1.26
458 453 -1.81
570 209 -1.21
152 241 0.52
223 202 9.26
130 379 2.21
86 297 9.09
70 512 9.96
412 398 6.64
560 225 8.52
391 195 7.52
571 525 1.66
491 263 5.28
123 179 1.29
574 427 0.55
257 322 4.29
439 518 8.25
195 101 6.28
229 253 7.92
290 49 3.6
478 235 2.0
543 224 9.03
462 530 6.8
187 442 -0.77
546 531 -0.11
247 332 1.62
424 23 8.15
55 46 -0.69
204 576 -0.55
362 10 8.06
124 529 0.69
459 337 7.45
407 463 -0.53
427 104 2.44
410 10 4.94
382 197 -1.0
102 560 2.86
253 398 6.79
111 215 0.67
141 136 8.35
254 380 -0.4
161 80 6.45
525 428 -0.61
156 306 5.71
61 94 4.55
509 561 7.5
322 360 2.42
177 382 7.96
109 204 9.61
423 567 8.81
464 511 1.94
249 388 4.68
244 177 7.53
41 23 6.9
223 171 4.85
243 250 0.08
106 269 8.34
230 275 2.33
99 45 4.15
19 337 -1.16
27 274 -1.76
81 73 1.77
204 208 4.99
110 225 0.42
68 220 1.97
114 147 0.28
374 282 4.4
525 407 1.2
538 27 5.46
505 0 -1.45
548 41 8.82
138 529 -0.3
231 153 -0.05
233 26 6.43
213 294 4.39
176 79 1.69
534 393 5.21
140 295 7.98
73 409 5.16
99 560 5.11
332 240 9.67
359 134 5.9
561 446 0.05
59 481 7.59
15 476 6.1
30 284 3.9
208 418 7.33
515 176 8.94
526 223 5.2
323 379 6.85
334 362 0.27
541 19 6.12
523 35 7.77
252 580 8.51
188 563 2.23
220 539 4.24
481 518 0.63
290 83 9.35
251 487 -1.39
566 560 0.79
275 110 3.19
482 63 5.68
348 570 7.68
316 42 2.27
368 217 -0.92
197 578 3.95
133 97 -1.1
34 12 1.38
32 82 4.62
467 398 3.57
91 149 0.28
264 461 0.24
564 64 6.13
69 301 0.18
554 96 4.41
564 438 2.85
106 378 4.34
2 510 -0.52
469 475 -1.15
443 208 1.79
289 212 -0.58
401 324 4.17
444 177 -0.5
236 139 0.27
16 178 -0.65
447 564 5.67
352 226 4.13
452 287 5.42
146 64 10.0
427 38 -0.78
188 459 2.89
54 318 6.2
107 444 -0.85
200 334 6.31
42 417 4.72
5 281 8.49
232 123 2.9
84 352 -1.13
198 405 -0.47
35 68 1.37
580 396 8.35
5 442 8.93
477 84 7.44
239 384 1.92
160 488 2.58
455 188 3.31
271 41 5.46
25 345 1.07
268 515 9.85
47 40 9.56
172 407 4.86
475 108 9.02
117 90 7.25
113 162 3.84
363 182 9.94
34 91 7.59
391 537 10.0
10 471 6.17
424 297 7.73
337 507 9.97
282 220 6.2
44 122 6.61
150 336 -1.08
213 25 2.87
318 560 0.19
343 579 3.8
59 519 8.23
140 375 -1.55
202 433 9.11
499 579 9.15
404 114 -0.06
12 527 -0.82
388 235 9.6
108 347 9.93
529 14 9.6
555 494 0.0
179 520 2.02
90 219 6.22
182 549 8.13
295 115 5.83
285 236 0.36
278 157 8.45
568 522 1.83
28 126 5.26
38 367 7.92
521 200 4.17
195 377 5.32
315 468 0.88
256 174 4.65
332 58 -1.69
356 489 5.14
103 259 5.74
532 530 5.92
304 448 3.25
341 336 -0.57
363 270 9.47
577 383 2.54
79 288 -0.76
471 196 -0.49
330 218 8.77
456 331 5.43
408 485 -1.24
572 344 2.8
580 39 2.08
282 411 6.88
291 307 8.4
384 381 9.38
261 123 9.89
258 170 8.4
411 185 9.88
396 345 8.49
549 503 8.24
80 443 1.44
4 162 4.97
354 169 -0.45
532 321 9.4
432 319 1.24
84 163 9.33
163 353 7.61
211 53 3.83
375 348 6.9
440 59 -1.23
56 480 -0.27
464 387 8.03
75 533 5.78
46 52 8.24
333 439 9.99
72 452 -0.43
420 145 7.97
436 550 3.07
339 197 -0.24
20 97 3.35
264 210 0.44
376 312 1.37
8 280 8.37
196 448 -0.55
307 322 7.38
174 286 2.25
56 384 0.02
222 36 4.07
463 404 7.61
261 485 3.86
223 109 9.56
133 201 8.46
469 115 -0.53
451 425 8.01
526 389 2.31
264 165 3.64
171 90 0.14
299 551 3.6
281 497 8.66
37 160 5.62
296 79 3.81
458 247 5.82
472 167 9.39
578 279 0.77
215 85 4.46
488 61 0.27
61 497 1.47
215 463 6.88
201 433 7.25
346 412 3.27
189 190 4.44
201 544 1.13
302 22 3.32
405 546 6.46
484 51 3.0
490 547 8.7
76 372 -1.44
159 354 6.77
376 369 2.64
367 124 7.89
503 194 1.36
342 520 8.52
363 347 2.88
480 495 7.76
61 309 1.41
386 337 6.59
301 44 6.16
298 552 6.74
171 571 0.24
398 190 4.74
430 368 -0.64
276 378 8.97
366 280 -0.3
189 316 5.5
3 391 3.34
187 90 7.58
233 395 3.83
248 295 -1.11
205 221 2.38
17 261 3.75
83 158 2.75
47 81 9.53
411 357 3.55
579 78 3.08
100 113 2.66
384 452 -1.17
401 467 7.72
128 362 6.62
43 577 4.01
402 118 0.27
208 538 6.33
341 532 8.84
288 82 3.72
283 389 5.52
532 189 7.59
340 385 4.27
14 476 8.47
431 47 6.14
203 148 5.16
334 248 5.96
249 355 2.0
502 406 3.09
5 283 7.08
101 254 -0.27
236 152 6.75
53 195 7.15
45 525 4.21
72 162 7.87
445 293 1.8
479 500 -1.64
533 486 -0.67
169 365 8.94
173 416 8.51
536 269 8.57
193 228 7.43
33 247 3.4
292 145 6.06
156 447 4.08
186 131 -1.22
476 297 0.93
137 118 0.41
268 548 1.9
438 565 6.52
201 160 0.79
130 392 8.09
144 8 8.13
166 43 6.82
419 376 6.06
501 37 5.51
535 350 0.02
389 174 3.66
391 343 6.36
119 79 8.98
370 254 2.61
82 33 9.95
466 100 3.46
150 177 5.56
253 289 1.79
116 523 8.91
205 461 5.88
116 276 7.18
93 380 9.18
304 374 1.19
171 496 6.0
298 22 8.71
562 574 0.75
435 66 5.72
403 86 -1.34
89 415 3.73
392 385 6.1
412 175 2.48
240 93 8.13
430 307 9.17
486 134 8.1
535 248 1.89
0 132 5.54
576 323 -1.41
313 77 6.07
538 551 1.15
262 379 1.7
521 2 4.8
278 431 3.49
258 203 5.45
389 57 9.35
125 120 2.49
263 291 1.76
365 15 6.89
60 310 4.92
441 556 3.06
246 434 5.34
203 176 9.08
485 146 3.87
557 325 -0.23
552 436 4.25
335 286 9.63
262 117 6.63
213 117 1.49
280 491 4.43
10 203 6.72
370 427 4.02
102 381 -1.28
12 38 1.41
474 266 6.02
195 573 3.39
568 111 7.89
176 47 -0.53
263 117 4.49
353 544 7.49
426 409 4.39
294 359 1.29
257 288 8.89
149 291 8.52
579 473 9.24
415 401 9.03
326 249 3.83
364 381 -0.06
132 350 4.1
241 338 8.61
527 106 9.66
336 243 6.36
205 151 7.19
183 475 9.99
364 291 5.48
528 557 1.7
472 84 6.74
520 153 7.88
558 164 7.02
460 480 -0.69
462 330 7.8
200 377 8.75
452 109 4.59
62 421 -1.99
214 569 6.85
580 142 1.2
37 301 5.83
437 149 5.37
314 333 6.0
398 86 -1.3
517 200 8.69
107 173 -1.58
109 478 3.5
465 263 -0.82
535 327 0.76
415 20 8.66
442 496 9.37
77 245 6.65
513 298 6.34
326 466 6.39
530 105 6.85
475 541 -0.42
278 80 0.27
331 3 6.31
333 420 9.18
324 90 6.19
284 198 2.85
525 367 1.96
238 206 3.55
285 120 6.98
494 351 8.39
320 368 -1.5
307 324 5.29
331 180 6.21
64 309 5.16
531 14 2.96
522 345 8.03
30 118 7.74
508 304 2.63
492 296 8.11
522 54 -1.75
513 119 -0.73
139 249 6.8
151 439 5.05
357 158 5.15
224 21 -0.33
67 314 3.94
226 222 9.29
295 434 3.8
258 574 2.59
173 252 2.22
542 271 3.56
463 125 0.25
15 382 3.56
367 576 2.54
494 526 3.18
94 342 8.04
284 22 -1.87
323 21 5.09
493 538 9.25
432 32 5.01
10 130 3.56
322 272 5.0
574 340 -1.27
170 469 9.15
197 27 -0.06
524 504 3.5
452 568 7.02
478 363 6.33
446 43 2.33
87 221 6.33
26 238 2.55
455 133 8.76
218 402 2.04
60 292 0.48
68 314 2.41
269 349 4.91
304 376 3.8
449 196 3.93
96 124 2.34
377 567 0.34
65 137 5.41
495 473 3.95
501 302 7.99
470 569 3.78
87 260 1.71
21 217 6.24
321 406 3.01
270 90 -0.8
281 572 5.25
242 64 -0.98
156 339 9.74
115 125 1.33
194 437 1.3
577 529 4.25
151 475 6.2
372 537 0.22
171 315 9.99
486 229 2.57
579 448 6.04
236 530 5.46
131 548 4.36
196 60 8.47
265 533 3.36
361 573 0.77
297 71 9.89
154 277 7.92
439 454 4.54
3 361 1.51
450 521 -1.09
2 460 4.9
38 232 5.6
118 398 7.7
313 320 -0.51
39 36 2.96
31 60 8.51
368 463 9.48
548 78 9.72
142 446 5.89
355 352 0.5
501 78 0.08
145 558 5.27
117 70 9.37
244 184 8.19
420 438 0.96
324 204 5.78
97 508 3.6
206 79 9.84
92 94 5.52
41 564 1.62
44 166 8.62
119 349 -0.81
396 130 8.49
227 21 8.47
547 115 4.85
569 35 2.32
71 65 -1.06
218 92 8.07
192 25 2.48
500 559 -0.59
135 349 2.04
315 150 3.9
378 72 7.69
351 472 9.39
468 178 -0.01
447 139 2.94
196 512 1.05
434 488 1.74
381 415 5.77
303 230 7.31
153 161 2.42
310 277 6.96
448 39 6.79
81 119 -0.19
202 544 6.8
216 130 1.61
25 109 5.64
324 388 4.15
485 192 4.95
572 111 -1.05
449 442 3.84
54 446 4.55
105 455 -1.85
526 294 6.75
459 33 9.5
327 534 -1.2
271 571 4.34
424 204 2.88
272 333 5.56
314 294 8.66
292 141 0.67
80 27 7.87
131 481 6.89
347 358 0.16
410 129 3.36
536 306 4.95
567 554 2.14
286 532 1.9
308 46 8.65
275 305 5.93
487 4 5.76
167 470 4.03
338 319 6.49
558 246 8.82
110 507 -1.34
22 251 -0.4
471 523 7.66
371 143 0.15
433 351 -1.85
279 193 6.21
400 67 2.97
11 543 -0.47
136 177 1.74
224 255 -1.98
404 265 6.66
67 413 3.19
5 87 6.61
342 575 3.89
550 159 -0.17
111 309 5.83
259 231 -0.46
214 336 -0.02
454 152 6.17
396 417 8.93
311 181 2.64
131 287 2.1
168 445 4.74
115 233 3.98
50 522 1.4
344 121 9.21
473 62 0.62
489 8 9.5
438 308 2.37
451 261 9.39
171 578 0.82
512 49 1.95
394 490 -1.32
413 402 4.72
93 182 9.43
389 497 -0.05
425 226 0.52
547 92 2.81
301 92 9.62
185 541 -0.04
495 78 6.0
52 548 6.03
222 254 0.52
18 167 1.49
386 557 7.72
311 5 9.33
159 133 9.08
386 281 6.65
524 256 -0.95
465 184 0.83
383 196 9.56
215 401 7.29
483 466 2.49
449 1 5.31
304 511 4.83
437 141 8.0
98 176 1.15
273 413 -1.28
402 339 -0.52
376 553 3.61
306 530 9.11
10 162 8.7
481 552 0.13
390 346 8.28
243 114 2.86
210 133 9.0
251 302 9.31
216 303 0.57
165 268 2.97
50 68 0.82
90 525 7.33
8 180 -1.45
56 252 0.76
291 87 1.63
138 293 2.48
111 406 6.02
565 4 8.04
405 364 9.94
254 412 -0.67
377 183 2.49
325 91 -0.26
192 312 8.69
378 557 4.58
132 103 7.84
328 451 1.15
133 142 -1.53
121 262 1.94
501 341 -0.17
525 326 3.84
13 424 6.82
181 113 5.44
105 289 0.55
358 517 3.31
209 522 5.27
262 462 -0.03
68 416 4.81
321 331 1.38
112 18 3.95
180 293 6.63
177 171 6.72
456 116 1.88
215 546 8.1
466 395 8.11
261 229 2.05
119 273 9.2
326 69 7.51
176 38 -0.79
565 468 3.86
369 166 4.92
228 216 7.85
571 30 1.28
402 343 -1.29
502 192 1.54
182 9 3.56
84 335 9.22
70 503 4.69
323 282 3.34
410 370 4.14
485 159 7.27
21 266 4.37
228 496 5.66
295 146 6.26
59 516 8.31
352 163 7.33
548 233 3.85
180 311 5.63
277 550 9.64
26 546 4.06
16 208 5.05
518 394 5.96
219 577 9.34
246 3 0.41
436 390 -0.02
448 479 -0.46
60 434 4.8
165 60 9.11
58 412 1.63
482 269 2.5
498 256 -1.01
512 331 6.17
497 107 9.47
493 56 6.65
539 314 3.13
433 327 0.86
552 234 6.76
506 95 4.33
378 334 0.32
286 401 9.96
212 102 1.24
539 513 1.83
190 213 6.05
203 15 2.92
550 146 3.58
211 81 0.42
385 525 6.82
516 240 0.93
242 201 8.04
445 395 8.71
305 48 9.31
24 461 7.81
293 283 -1.83
369 229 6.84
471 524 1.62
104 454 3.17
251 393 1.78
497 563 2.15
95 429 5.2
127 499 0.74
319 449 6.59
73 523 -0.39
265 228 7.98
515 211 4.41
118 396 1.68
399 206 -0.53
120 207 1.06
15 194 4.83
268 121 6.34
304 112 8.76
429 438 8.04
560 26 4.05
22 184 7.06
188 386 1.44
403 161 5.56
532 268 7.55
297 39 9.93
149 298 3.85
43 302 1.55
5 183 5.62
578 164 0.94
240 390 3.4
466 203 5.58
468 171 8.39
150 69 8.22
308 187 0.37
181 261 9.14
168 394 8.31
85 332 4.18
382 291 -1.2
438 159 1.6
55 373 -1.77
364 484 3.01
376 228 6.42
55 283 3.36
110 389 2.0
207 138 3.06
61 528 9.35
573 282 6.69
543 436 4.4
299 119 -0.02
248 301 9.94
437 18 2.66
64 422 7.81
124 410 5.83
337 514 3.7
553 321 1.6
56 263 2.71
539 468 4.98
88 568 7.76
503 562 6.5
51 7 -1.41
418 513 -1.99
76 328 7.05
226 409 9.1
414 419 0.33
398 103 -1.21
556 252 7.36
96 285 6.43
93 551 3.85
142 455 5.83
300 72 9.5
196 422 4.08
474 532 -1.4
74 458 2.42
23 199 0.15
504 536 5.75
87 131 -0.07
408 428 3.19
119 219 0.98
56 203 1.75
191 464 3.71
164 16 8.44
287 92 5.11
463 520 2.1
318 107 8.06
246 167 7.85
