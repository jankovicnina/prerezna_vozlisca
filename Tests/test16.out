27.478212979999114
